packages/yoroi-extension/app/App.js

```diff
@@ -126,38 +128,79 @@ class App extends Component<Props, State> {
 
     return (
       <div style={{ height: '100%' }}>
-        <LayoutProvider layout={currentTheme}>
-          <ThemeProvider theme={muiTheme}>
-            <CssBaseline />
-            {globalStyles(muiTheme)}
-            <ThemeManager cssVariables={themeVars} />
-            {/* Automatically pass a theme prop to all components in this subtree. */}
-            <IntlProvider {...{ locale, key: locale, messages: mergedMessages }}>
-              {this.getContent()}
-            </IntlProvider>
-          </ThemeProvider>
-        </LayoutProvider>
+        <QueryClientProvider client={queryClient}>
+          <MetricsProvider storage={metricsStorage} metrics={amplitudeClient}>
```

Comment:
`@yoroi/metrics` requires a provider, any package specifics are created and injected, any manipulation *must happen* right before it.